modules/llrt_assert/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use llrt_utils::module::ModuleInfo;
+use rquickjs::{
+    module::{Declarations, Exports, ModuleDef},
+    prelude::Opt,
+    Ctx, Exception, Function, Result, Type, Value,
+};
+
+fn ok(ctx: Ctx, value: Value, message: Opt<Value>) -> Result<()> {
+    match value.type_of() {
+        Type::Bool => {
+            if value.as_bool().unwrap() {
+                return Ok(());
+            }
+        },
+        Type::Float | Type::Int => {
+            if value.as_number().unwrap() != 0.0 {
+                return Ok(());
+            }
+        },
+        Type::String => {
+            if !value.as_string().unwrap().to_string().unwrap().is_empty() {
+                return Ok(());
+            }
+        },
+        Type::Array | Type::Object => {
+            return Ok(());
+        },
+        _ => {},
+    }
+
+    if let Some(obj) = message.0 {
+        match obj.type_of() {
+            Type::String => {
+                let msg = obj.as_string().unwrap().to_string().unwrap();
+                return Err(Exception::throw_message(&ctx, &msg));
+            },
+            Type::Exception => return Err(obj.as_exception().cloned().unwrap().throw()),
+            _ => {},
+        };
+    }
+
+    Err(Exception::throw_message(
+        &ctx,
+        "The expression was evaluated to a falsy value",
+    ))
```

Comment:
Ideally we should ha an `AssertionError` but that can be added later. It might be good to show wha the value was by coercing it to string